src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/samples/Sample1_HelloWorld.cs

```diff
@@ -16,8 +16,8 @@ public partial class ConfigurationSamples
         public async Task HelloWorld()
         {
             // Retrieve the connection string from the configuration store. 
-            // You can get the string from your Azure portal or using Azure CLI.
-            var connectionString = Environment.GetEnvironmentVariable("AZ_CONFIG_CONNECTION");
+            // You can get the string from your Azure portal.
+            var connectionString = Environment.GetEnvironmentVariable("APP_CONFIG_CONNECTION");
```

Comment:
You need to change the internal build pipeline definition to set the correct environment variable.

Reply:
Good catch. Thank you. 
Pipeline updated and CI internal updated too